actionpack/test/controller/log_subscriber_test.rb

```diff
@@ -258,6 +259,18 @@ def test_with_fragment_cache
     @controller.config.perform_caching = true
   end
 
+  def test_with_fragment_cache_when_log_disabled
+    @controller.config.perform_caching = true
+    @controller.enable_fragment_cache_logging = false
```

Comment:
Sorry, I don't really understand the situation you are saying. Can you explain more?